roles/openshift_logging/tasks/annotate_ops_projects.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- oc_obj:
+    state: list
+    kind: project
+    name: "{{ item }}"
+  with_items: "{{ __default_logging_ops_projects }}"
+  register: __logging_ops_projects
+
+- name: Annotate Operations Projects
+  oc_edit:
+    kind: ns
+    name: "{{ item.item }}"
+    separator: '#'
+    content:
+      metadata#annotations#openshift.io/logging.ui.url: "{{ openshift_logging_kibana_ops_hostname }}"
```

Comment:
Should this be `openshift.io/logging.ui.hostname` if it's not actually a URL?

Reply:
updated.